src/plugins/intel_npu/src/plugin/src/metrics.cpp

```diff
@@ -121,6 +123,24 @@ uint32_t Metrics::GetDriverExtVersion() const {
     return _backends->getDriverExtVersion();
 }
 
+uint32_t Metrics::GetSteppingNumber(const std::string& specifiedDeviceName) const {
+    const auto devName = getDeviceName(specifiedDeviceName);
+    auto device = _backends->getDevice(devName);
+    if (device) {
+        return device->getSubDevId();
+    }
+    OPENVINO_THROW("No device with name '", specifiedDeviceName, "' is available");
```

Comment:
What is happening in the case of offline compilation here?

Reply:
added offline verification at init stage, thanks!